plugins/main/bridge/bridge.go

```diff
@@ -465,8 +461,36 @@ func cmdAdd(args *skel.CmdArgs) error {
 			if err := ipam.ConfigureIface(args.IfName, result); err != nil {
 				return err
 			}
+			return nil
+		}); err != nil {
+			return err
+		}
+
+		// check bridge port state
+		retries := []int{50, 500, 1000, 1000}
+		for idx, sleep := range retries {
+			hostVeth, err := netlink.LinkByName(hostInterface.Name)
+			if err != nil {
+				return err
+			}
+			if hostVeth.Attrs().OperState == netlink.OperUp {
+				break
+			}
+
+			if idx == len(retries)-1 {
```

Comment:
If we try to escape from here, we only try 4 times and need 3 intervals, so I think we should sleep in the beginning of the loop and let `retries := []int{0,50,500,1000,1000}` , so we will make full use of `retries`.

Reply:
@mars1024   add it.

Reply:
Sorry, I think sleep should be the first step of the loop... just like
```golang
retries := []int{0, 50, 500, 1000, 1000}
for idx, sleep := range retries {
  time.Sleep(sleep * time.Millisecond)

  (check host)

  if idx == (len(retries) - 1) {
    fail
  }
}
```

Reply:
sleep 0 before   do something ?

Reply:
yes, 0 means no sleep in the first time, then we will have 5 tries and 4 non-zero intervals as expectation.

Reply:
@mars1024 ,  have changed.